### PR TITLE
Add misspell: train, transaction, transition, trigger, troubleshoot

### PR DESCRIPTION
**Pull request by vikivivi**
New entry words: throttled, tracked, trackers, train, trained, trains, transients, transitioning, troubleshoot
* Typical code search:
  * https://github.com/search?q=trainig&type=code
  * https://github.com/search?q=trainling&type=code
  * https://github.com/search?q=tranistion&type=code
  * https://github.com/search?q=transaciton&type=code
  * https://github.com/search?q=trian&type=code
  * https://github.com/search?q=triggerring&type=code
  * https://github.com/search?q=trrigers&type=code

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -28571,11 +28592,20 @@ trnsfered->transferred
 trnsfers->transfers
 troling->trolling
 trottle->throttle
+trottled->throttled
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt:28595]: also `trotted`

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -28571,11 +28592,20 @@ trnsfered->transferred
...
+trottling->throttling
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt:28596]: also `trotting`

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -28237,9 +28246,15 @@ tranmitting->transmitting
 tranparent->transparent
 tranparently->transparently
 tranport->transport
+tranports->transports
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt:28249]: Would you mind adding entries tor:
`tranported->transported`  
`tranporting->transporting`

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -28223,6 +28228,10 @@ tranformed->transformed
 tranforming->transforming
 tranforms->transforms
 tranient->transient
+tranients->transients
+tranistion->transition
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt:28232]: Please add `tranistioned->transitioned`

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -28194,7 +28194,12 @@ trageting->targeting
 tragets->targets
 traiing->trailing, training,
 trailling->trailing
+trainig->training
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt:28197]: Please add ` trainigs->trainings`

**Review by luzpaz — CHANGES_REQUESTED** [review]

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (vikivivi) [reply codespell_lib/data/dictionary.txt:28595]: `trotted` is a valid word and valid spelling. Verified with Cambridge, Collins, Dictionary.com, Merriam-Webster, macmillan, Oxford Learners, aspell, hunspell, ispell, etc

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (vikivivi) [reply codespell_lib/data/dictionary.txt:28596]: `trotting` is a valid word and valid spelling. Verified with Cambridge, Collins, Dictionary.com, Merriam-Webster, macmillan, Oxford Learners, aspell, hunspell, ispell, etc

**Comment by vikivivi** [comment]
@luzpaz Done.

**Comment by luzpaz** [comment]
Yikes, you went too far. lol

**Review by peternewman — COMMENTED** [review]
Please can we fix the merge conflicts

**Comment by vikivivi** [comment]
@peternewman I rebase to resolve conflicts with squash commits.

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -28498,16 +28498,32 @@ traingulates->triangulates
 traingulating->triangulating
 traingulation->triangulation
 traingulations->triangulations
+trainig->training
+trainigs->trainings
+trainling->training
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:28503]: Suggest trailing too.

File: codespell_lib/data/dictionary.txt [hunk 7]
```diff
@@ -28498,16 +28498,32 @@ traingulates->triangulates
...
+trainng->training
+trainngs->trainings
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:28505]: I thought it seemed odd, so suggest both plural and singular:
https://www.englishbusiness.com/blog/training-or-trainings-what-is-the-correct-plural-form/

You may also want to add `trainings->training` to the US to GB dictionary.

File: codespell_lib/data/dictionary.txt [hunk 8]
```diff
@@ -28498,16 +28498,32 @@ traingulates->triangulates
...
+traked->tracked
 traker->tracker
+trakers->trackers
 traking->tracking
-traling->trailing, trialing,
-trama->trauma
+tral->trail, trial,
+traled->trailed, trialed, trawled, traced, traded, traveled,
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:28511]: traced/traded seem rather unlikely to me, but it doesn't really matter.

File: codespell_lib/data/dictionary.txt [hunk 9]
```diff
@@ -28498,16 +28498,32 @@ traingulates->triangulates
...
+traling->trailing, trialing, trawling, tracing, trading, traveling,
+tralled->trailed, trialled, trilled, trolled, trawled, travelled,
+tralling->trailing, trialling, trilling, trolling, thralling, travelling,
+trals->trails, trials,
+trama->trauma, trams,
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:28516]: Suggest tram singular too.

File: codespell_lib/data/dictionary.txt [hunk 10]
```diff
@@ -28840,13 +28869,17 @@ tressle->trestle
 treting->treating
 trgistration->registration
 trhe->the
+trian->train
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:28872]: Could suggest trial

File: codespell_lib/data/dictionary.txt [hunk 11]
```diff
@@ -28882,11 +28917,20 @@ trnsfered->transferred
...
+troublshoot->troubleshoot
 troublshooting->troubleshooting
 troughput->throughput
 trought->through
 troup->troupe
 troups->troupes, troops,
+trriger->trigger
+trrigered->triggered
+trrigering->triggering
+trrigers->triggers
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:28933]: It probably makes sense to add these typos with the correct number of g's too.

**Review by peternewman — CHANGES_REQUESTED** [review]
Some comments, thanks for fixing the conflict.

**Comment by bl-ue** [comment]
ping @vikivivi can I finish this for you?

**Comment by vikivivi** [comment]
@bl-ue Agreed. You may go ahead. I am just busy at the moment.

**Comment by bl-ue** [comment]
No problem whatsoever — I happen to have the time right now :)

**Review by bl-ue — APPROVED** [review]

**Comment by bl-ue** [comment]
@peternewman I've implemented all of your suggestions; PTAL.

**Comment by bl-ue** [comment]
Hmm, I'm not sure what the errors mean — @peternewman?

**Review by bl-ue — APPROVED** [review]

**Pull request merged**